src/main/webapp/resources/js/pages/UsersPage.jsx

```diff
@@ -175,4 +176,7 @@ function UsersPage() {
   );
 }
 
-render(<UsersPage />, document.querySelector("#react-root"));
```

Comment:
This is kind of ugly.  Can you move the \`UsersTable\` into it's own file, in the components directory (follow the naming in there for directory names).  Then create a `AdminUsersPage` that imports that.  The benefit is that it prevents checking the page variable if the user is an admin, you know if they are in the admin section that they must be one.  You can then pass that in as a param into the `UsersTable` component.